src/cli/bin.ts

```diff
@@ -30,26 +31,43 @@ yargs
         demandOption: false,
         desc: "A list of keys of ActiveUI 4 widget plugins that should be removed during the migration.",
       });
+      args.option("pivot-input-path", {
+        alias: "p",
+        type: "string",
+        demandOption: false,
+        desc: "The path to the JSON export of the ActiveUI 4 /pivot folder",
```

Comment:
The `/pivot` folder is not bound only to ActiveUI 4.
We just happened to save some content in there in ActiveUI 4.
```suggestion
        desc: "The path to the JSON export of the /pivot folder on the content server",
```